stories/topics/proc-saas-set-up-public.adoc

```diff
@@ -22,12 +32,26 @@ As an AWS customer you can search for and subscribe to {SaaSonAWS} from the AWS
 .. Create or sign into your Red{nbsp}Hat account.
 .. Link your AWS account to the Red{nbsp}Hat account.
 . This then redirects you to the *Provision environment* page on the Red{nbsp}Hat Hybrid Cloud Console. Here you can start configuring your environment. 
-. Enter your AWS account ID.
+. Enter your AWS account ID and click btn:[Verify].
 +
 This account ID must be the account ID that purchased the offer from the AWS marketplace. 
-The system will not recognize associated or nested accounts.
-. Once your AWS ID is validated click btn:[Next].
+The system does not recognize associated or nested accounts.
+. After your AWS ID is validated click btn:[Next].
 . Select your desired region.
 . Click btn:[Provision].
 
-This redirects you to the *Instances* page which displays all the details of the instance you created. From here you can confirm if your instance is *Assigned* or still in a *Pending* state.
\ No newline at end of file
+This redirects you to the *Environment details* page of the *Instances* tab which displays all the details of the instance you created.
+From here you can view if your instance is *Ready* or still in a *Provision in progress* state.
+
+Provisioning your environment can take up to two hours.
+When your environment is ready a confirmation email is sent to the email address linked to your account. 
+
+[IMPORTANT]
+====
+You need to update your password as soon as possible.
+Your confirmation email includes a link that shows your admin password.
+Use this password to log-in and change your password.
+This link can only be accessed once, so ensure you are ready before clicking it. 
+
+In the event the email is not received or initial admin password expires, you need to submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.
```

Comment:
```suggestion
In the event the email is not received or initial admin password expires, you need to submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.
```
If you did not receive the email or the initial admin password has expired, you must submit a link:https://access.redhat.com/support[support ticket] and the Red{nbsp}Hat team will assist you with next steps.